mule-migration-tool-e2e-tests/src/test/resources/e2e/batch/batch1/output/src/main/mule/mule-config.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:batch="http://www.mulesoft.org/schema/mule/batch" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mulesoft.org/schema/mule/batch http://www.mulesoft.org/schema/mule/batch/current/mule-batch.xsd http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd">
+
+    <flow name="dispatcher">
+        <flow-ref name="#[vars.compatibility_inboundProperties['batchName']]" />
+
+        <set-payload value="10" />
+
+    </flow>
+
+    <flow name="concurrent">
+        <set-payload value="'hey apple!'" />
+
+        <batch:job jobName="concurrent" schedulingStrategy="ROUND_ROBIN">
+            <batch:history>
+                <batch:expiration maxAge="10" ageUnit="SECONDS" />
+            </batch:history>
+            <batch:process-records>
+                <batch:step name="firstStep" acceptExpression="#[payload == 'hey apple!']">
+                    <batch:aggregator size="10">
+                        <set-payload value="#[payload[0]]" />
+                    </batch:aggregator>
+                </batch:step>
+                <batch:step name="secondStep">
+                    <flow-ref name="upperCasePipe" />
+                    <logger message="something" />
+                </batch:step>
+            </batch:process-records>
+            <batch:on-complete>
+                <logger message="something" />
+            </batch:on-complete>
+        </batch:job>
+
+    </flow>
+
+    <sub-flow name="upperCasePipe">
+        <set-payload value="#[mel:payload + '|' + payload.toUpperCase()]">

```

Comment:
maybe its a good case to analyze why the expression is not migrated?

Reply:
I'll include this as part of the task to analyze mel expressions

Reply:
Sounds good